### PR TITLE
[JENKINS-28543] Strengthen test to cover buildWithParameters

### PR DESCRIPTION
**Pull request by jglick**
[JENKINS-28543](https://issues.jenkins-ci.org/browse/JENKINS-28543)

@reviewbybees

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
:bee:

**Comment by tfennelly** [comment]
:bee:

File: src/test/java/org/jenkinsci/plugins/build_token_root/BuildRootActionTest.java [hunk 1]
```diff
@@ -64,38 +71,40 @@
...
         assertEquals(0, p.getBuilds().size());
         Page page = wc.goTo("buildByToken/build?job=" + p.getFullName() + "&token=secret&delay=0sec", null);
-        j.waitUntilNoActivity();
+        assertCreated(page);
         assertEquals(1, p.getBuilds().size());
-        assertEquals(HttpStatus.SC_CREATED, page.getWebResponse().getStatusCode());
-        assertTrue(page.getWebResponse().getResponseHeaderValue("Location").contains("/queue/item/"));
         page = wc.goTo("buildByToken/build?job=" + p.getFullName() + "&token=secret&delay=0sec", null);
-        j.waitUntilNoActivity();
+        assertCreated(page);
```
Comment (amuniz) [inline src/test/java/org/jenkinsci/plugins/build_token_root/BuildRootActionTest.java]: Why are you doing the same check twice and consecutive?

Comment (jglick) [inline src/test/java/org/jenkinsci/plugins/build_token_root/BuildRootActionTest.java]: Do not recall now. I suppose just to verify that each web request produces a build. Anyway that existed before this PR.

**Pull request merged**